cli/src/plugins/javadoc/index.ts

```diff
@@ -275,6 +275,26 @@ function makeClassDocPageBody(args: MakeSectionArgs): Node[] {
         ),
     }),
 
+    ...makeSection({
+      ...args,
+      depth,
+      title: "Enum Constant Summary",
+      shouldMakeSection: () => doc.enumConstants.length !== 0,
+      makeBody: () =>
+        makeTable(
+          ["Enum Constant", "Description"],
+          doc.enumConstants.map((doc) => [
+            [
+              project.linkToEntity(
+                doc.qualifiedName + "_" + doc.name,
```

Comment:
Ensure this matches the other one. I would do this by creating a separate function to "get enum canonical name" from a doc.

Reply:
After poking at this more, I realize I was conflating a couple of things when I was trying to get the canonical name, and I believe it's just `doc.qualifiedName`. Presuming since it's a single param we would just reference it directly here, vs. creating a function to get it?

Reply:
As long as doc.qualifiedName is unique across the project (it should be), then that's fine.